docs/UserGuide.md

```diff
@@ -130,8 +129,10 @@ Displays all grading components and their respective weights for a class.
     - `list_component`
 
 ---
-
-
+## Notes:
+- Matric Number (`i/` argument) is case insensitive. Therefore, `A1234567X` is the same as `a1234567x`. Matric numbers 
+will be converted to uppercase for storage. 
```

Comment:
Nice note on the storage case, are we standardising upper or lower case for the storage and display of other information such as Students?

Reply:
We should but I haven't checked.

Reply:
Okay, as we discussed offline, we will not be standardising the case for storage and display, they will simply follow the input case given

Reply:
Just leaving this here, this is our agreement from our meeting:

Display and Storage:
- matricNumber --> convert to UPPERCASE
- Other data --> keep as original case

Matching and Finding:
- Set to UPPERCASE before comparison